subprojects/core-api/src/main/java/org/gradle/api/artifacts/transform/TransformParameters.java

```diff
@@ -25,4 +25,5 @@
  */
 @Incubating
 public interface TransformParameters {
+    interface None extends TransformParameters {}
```

Comment:
This should be a final class instead with a private constructor to avoid any confusion about inheritance and instantiation.

Reply:
I changed it to a final class with a private constructor.